www/main/navigator/search/components/SearchContent.jsx

```diff
@@ -72,6 +73,7 @@ const SearchContent = () => {
   };
 
   const loadMoreSearchResults = useCallback(() => {
+    if (akhandpatt) return;
```

Comment:
@amitojsingh veerji Any reason why we are we restricting search results in akhand path mode to 20?

Reply:
fixed